src/components/navBar/navBar.module.scss

```diff
@@ -1,35 +1,122 @@
-$offset: 10px;
-$thickness: 3px;
+.wrapper {
```

Comment:
Can we have the CSS nested? We are using SASS nesting in other stylesheets.
https://sass-lang.com/documentation/style-rules#advanced-nesting